osmosis/assetlist.json

```diff
@@ -16290,8 +16290,224 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
       },
       "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
+        }
+      ]
+    },
+    {
```

Comment:
all the other images have no padding, but the qatom images have a bit of padding. Can you please provide an image without padding so the sizing on displays will be consistent?

Reply:
Done @JeremyParish69